nsx-alb/30.x/README.md

```diff
@@ -0,0 +1,41 @@
+# VMware NSX Advanced Load Balancer STIG Compliance and Automation
```

Comment:
The product name has changed to VMware Avi Load Balancer. Please change the folder name to "avi" and the product name accordingly as well.